ui-metadata/onfido/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+  "common": {
+      "configProperties": [
+        {
+          "name": "token",
+          "displayOrder": 1,
+          "hint": "The API token generated via the Onfido dashboard",
+          "label": "API Token",
+          "type": "password",
+          "required": true,
+          "isSecret": true,
+          "dataComponentId": "idvp-edit-config-settings-token",
+          "maxLength": 100
+        },
+        {
+          "name": "webhook_token",
+          "displayOrder": 2,
+          "hint": "The Webhook token generated via the Onfido dashboard",
```

Comment:
```suggestion
          "hint": "The webhook token generated via the Onfido dashboard.",
```

Reply:
Add full stops to each hint.

Reply:
Resolved by 188a2b52e5a9ce9c7c137420c2c5d43f527ed7f4